abstract.md

```diff
@@ -0,0 +1,57 @@
+Redesigning the Numba Frontend for Rapidly Changing Python Bytecode
+-------------------------------------------------------------------
+
+The advent of the new Python release policy has created a number of challenges
+for the Numba project. Numba uses the Python bytecode as an interface to
+Just-In-Time compile Python functions. Unfortunately the bytecode was never
+designed to be stable and was never intended to serve as a public interface. As
+such the Bytecode specification is subject to constant change with every new
+Python minor version, new bytecodes are regularly introduced and the behaviour
+of existing byetcodes themselves may change in subtle, sometimes
+non-documented ways (examples?). Effectively the Python bytecode is largely not
+designed for humans or any systems other than the CPython interpreter to
+consume and as such can only be described as "vomit on his sweater already,
+mom's spaghetti.".
+
+The challenge that Numba faces is that significant work is needed every year in
+order to support the latest Python minor version. Until recently this has been
+a cacophony of largely unprincipled hacks, workarounds and heuristics in order
+to support new Python versions. In this talk I will introduce a new,
+principled approach to regularize (or sanitize) Python bytecode. The ideas used
+here are inspired by recent publications in theoretical computer science on the
+topics of Regionalized State Value Dependency Graphs or RVSDGs as they are
+commonly known. While we don't transfer directly to RVSDGs, we have found that
+an intermediary representation introduced in (ref:: paper) may be sufficient for
+Numba's needs. The basic idea is to take the bytecode spaghetti, which
+effectively describes a Control Flow Graph (CFG) of Basic Blocks (BBs) and
+transform this into a sane format. Some features of this format are the clear
```

Comment:
"sane" 😏 
maybe "structured" for the final version

Reply:
I have submitted a second draft, where I re-wrote several things.